src/test/ensemble_flow_custom_node_tests.cpp

```diff
@@ -1566,7 +1495,13 @@ class EnsembleFlowCustomNodeLoadConfigThenExecuteTest : public EnsembleFlowCusto
     }
 
     void loadConfiguration(const char* configContent, Status expectedStatus = StatusCode::OK) {
-        createConfigFileWithContent(configContent, configJsonFilePath);
+        std::string ovmsConfig = std::string(configContent);
+
+        std::string newDir = getGenericFullPathForBazelOut("/ovms/bazel-bin/src");
+        std::regex regexPattern(R"(/ovms/bazel-bin/src)");
```

Comment:
todo: use adjust from test_utils

Reply:
done

Reply:
?

Reply:
Using adjust function is not enough in this test to perform replacement so I do it manually